src/content/blog/releases/release-sdk-java-7.1.0.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Serverless Workflow Java SDK 7.1.0
+author: Francisco Javier Tirado Sarti
+date: 2025-04-29
+description: >
+   Heads up, community! We have released a new Java SDK version compatible with Specification 1.0.0.
+---
+
+## Announcing the Release of Serverless Workflow Java SDK 7.1.0
+
+Hello, Serverless Workflow community!
+
+We're excited to announce the release of the latest Java SDK, version 7.1.0, which is now compatible with the upcoming Serverless Workflow Specification 1.0.0 and includes a reference implementation. The reference implementation provides a ready-to-use runtime that supports the Serverless Workflow Specification. It includes a workflow execution engine, validation utilities, and illustrative examples to help you quickly test and deploy your workflows. 
+
+
+## Java SDK example
+
+You can dive right in using [one of the examples](https://github.com/serverlessworkflow/specification/tree/main/examples) available in the specification repository. Here's a quick example to get you started:
+
+To use the SDK, simply add it as a dependency to your Maven project:
+
+
+```xml
+<dependency>
+    <groupId>io.serverlessworkflow</groupId>
+    <artifactId>serverlessworkflow-api</artifactId>
+    <version>7.0.0</version>
```

Comment:
```suggestion
    <version>7.1.0.Final</version>
```
You can add this link to the blog too:
https://github.com/serverlessworkflow/sdk-java/releases/tag/7.1.0.Final